### PR TITLE
feat(types): upgrade types according to Polymesh 3.0.1

### PR DESCRIPTION
**Pull request by VictorVicente**

### CODE REVIEW

File: src/utils/conversion.ts [hunk 1]
```diff
@@ -1504,6 +1504,10 @@ export function claimToMeshClaim(claim: Claim, context: Context): MeshClaim {
       );
       break;
     }
+    case ClaimType.InvestorUniquenessV2: {
+      value = tuple(stringToCddId(claim.cddId, context));
```
Comment (monitz87) [inline src/utils/conversion.ts:1508]: Why a tuple?

**Review by monitz87 — CHANGES_REQUESTED** [review]

File: src/utils/conversion.ts (hunk 1 above)
Comment (VictorVicente) [reply src/utils/conversion.ts:1508]: It shouldn't be a tuple

**Review by monitz87 — APPROVED** [review]

**Pull request merged**